src/main/java/org/folio/edge/patron/PatronHandler.java

```diff
@@ -151,6 +153,21 @@ public void handlePlaceItemHold(RoutingContext ctx) {
             t -> handleProxyException(ctx, t)));
   }
 
+  public void handlePatronRequest(RoutingContext ctx) {
+    if (ctx.body().asJsonObject() == null) {
+      badRequest(ctx, MSG_HOLD_NOBODY);
```

Comment:
Please change the error message specific to this api

Reply:
fixed